package.json

```diff
@@ -176,6 +176,7 @@
     "pack-hdf-converters:win32": "pack-hdf-converters.bat",
     "pack-hdf-converters:darwin:linux": "./pack-hdf-converters.sh",
     "dev": "run-script-os",
+    "dev:run": "bin\\run",
```

Comment:
// ??

Reply:
dud,e give a break, doing testing, not final.